pwa/src/apiService/resources/openWoo.ts

```diff
@@ -13,7 +13,13 @@ export default class OpenWoo {
   }
 
   public getAll = async (filters?: IFiltersContext): Promise<any> => {
-    const { data } = await this._send(this._instance, "GET", `/openWOO?extend[]=all${filtersToQueryParams(filters)}`);
+    let url = `/openWOO?extend[]=all${filtersToQueryParams(filters)}`;
+
+    if (process.env.GATSBY_OIDN_NUMBER && process.env.GATSBY_OIDN_NUMBER !== " ") {
+      url += `&oidn=${process.env.GATSBY_OIDN_NUMBER}`;
+    }

```

Comment:
Depending on my earlier comment about making the ODIN-number required, we might be able to remove this.